<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_Main</class>
 <widget class="QMainWindow" name="Window_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(30, 30, 30)

}
QLabel{
color: rgb(200, 200, 200);

}

QPushButton {
background-color:rgb(40, 40, 40);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 10px;
padding: 1px;
}
QPushButton:hover {
background-color:rgb(20, 115, 230);
	color: rgb(200, 200, 200);
border: 1px solid rgb(20, 115, 230);
}
QPushButton:pressed {
background-color:rgb(18, 100, 200);
color: rgb(200, 200, 200);
border: 1px solid rgb(18, 100, 200);
}
QPushButton:disabled {
background-color:rgb(40, 40, 40);
color: rgb(100, 100, 100);
border: 2px solid rgb(50, 50, 50);
border-radius: 10px;
}

QLineEdit {
background-color:rgb(30, 30, 30);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 10px;
padding: 1px;
}

QLineEdit:focus {
background-color:rgb(30, 30, 30);
color: rgb(200, 200, 200);
border: 2px solid rgb(20, 115, 230);
border-radius: 10px;
padding: 1px;
}

QProgressBar {
background-color:rgb(30, 30, 30);
border: 2px solid rgb(80, 80, 80);
border-radius: 6px;
color:rgb(200, 200, 200);
text-align: center;
}
QProgressBar::chunk {
background-color:rgb(20, 115, 230);
border: 2px solid rgb(20, 115, 230);
border-radius: 1px;
}

QGroupBox {
color: rgb(200, 200, 200);
background: rgb(30, 30, 30);
border: 1px solid rgba(80, 80, 80, 255);
border-radius: 2px;
} 

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpbox_board">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>381</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string> Board </string>
    </property>
    <widget class="QLabel" name="lbl_grid_5">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border: 1px solid rgb(80, 80, 80);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_grid_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border: 1px solid rgb(80, 80, 80);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_grid_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>19</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border: 1px solid rgb(80, 80, 80);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_grid_0">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>19</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border: 1px solid rgb(80, 80, 80);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_grid_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border: 1px solid rgb(80, 80, 80);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_grid_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border: 1px solid rgb(80, 80, 80);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_grid_1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>19</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border: 1px solid rgb(80, 80, 80);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_grid_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border: 1px solid rgb(80, 80, 80);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_grid_8">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>260</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border: 1px solid rgb(80, 80, 80);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_game_reset">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>370</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_game_scan">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>370</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Force Scan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_game_start">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_box_anti_cheat">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>410</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(227, 227, 227);</string>
     </property>
     <property name="text">
      <string>Anti Cheat</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_box_auto_scan">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>410</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(227, 227, 227);</string>
     </property>
     <property name="text">
      <string>Auto Scan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_about">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>410</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpbox_live">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string> Preview </string>
    </property>
    <widget class="QLabel" name="lbl_video_live">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>216</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border: 1px solid rgb(80, 80, 80);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Select camera ID and press start to connect webcam.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_toggle_preview">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>320</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_cam_id">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera ID :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_calibrate">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibrate Board</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_fps">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>15</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FPS :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_board">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>250</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0);
border: 1px solid rgb(80, 80, 80);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_hands">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>219</y>
       <width>379</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(34, 255, 0, 150);
color: rgb(67, 67, 67);</string>
     </property>
     <property name="text">
      <string>Hand Detected</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>21</width>
      <height>421</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>821</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(200, 200, 200);
background-color:rgb(10, 10, 10);
border: 4px solid rgb(10, 10, 10);
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
